MIGRATING.md

```diff
@@ -2,6 +2,48 @@
 
 This library uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we don't make breaking changes unless the major version changes!
 
+## Migrating `.widgetrc` to `.widgtrc.js`
+
+The rule of thumb is to create a javascript file that `module.exports` the `widgetOptions` in your `.widgetrc` file.
```

Comment:
Probably important to callout the the previous `.widgetrc` file is now **removed**.

```suggestion
The existing `.widgetrc` file used to configure the Widget has been **removed**. Simply rename the existing file to `.widgetrc.js` export the contents:
```